### PR TITLE
Update assignment1/two_layer_netwrok

### PR DESCRIPTION
**Pull request by jiunbae**
번역이 힘든 부분이 있어 도움을 청합니다.
- respect to the variables 
- On the other hand, with a very large model we would expect to see more overfitting, which would manifest itself as a very large gap between the training and validation accuracy.

최대한 원본 의미를 살리기위해 고유명사 취급 할 수 있는 것들은 번역하지 않았는데 glossary에 추가해야할까요?

### CODE REVIEW

**Comment by myungsub** [comment]
1. 손실의 그라디언트와 respect to the variables `W1`, `b1`, `W2` and `b2`
   --> `W1`, `b1`, `W2`, `b2` 변수들에 대한 손실함수의 그라디언트
2. 반면에, 매우 큰 모델을 사용한다면 overfitting이 발생할 수 있는데, 이 경우 학습 정확도와 검증 정확도 사이에 매우 큰 차이가 나는 것을 확인할 수 있을 것이다.

이 정도로 번역하면 어떨까 싶습니다. 몇가지만 첨언하자면, (개인적으로는) '훈련'이라는 단어보다 '학습시키다'는 단어가 더 자연스럽게 느껴지고, '망 학습시키기' --> '신경망 학습시키기' 가 더 낫지 않을까 의견 여쭤봅니다!
최근에 다른 일로 바빴어서 신경을 많이 못 썼네요ㅠ 끝까지 잘 마무리될 수 있도록 화이팅입니다 :)

**Comment by jiunbae** [comment]
추가해봤습니다. 

훈련 -> 학습도 어감이 더 괜찮은것 같아서 다 바꾸었습니다.

**Comment by myungsub** [comment]
넵 감사합니다. 수고하셨습니다!
